README.md

```diff
@@ -30,12 +46,22 @@ To get a clone of the project, run `git clone https://github.com/white3d/Rails-R
 cd /Rails-React-Doctor-Appointment-App
 ```
 
+```
+rake db:migrate 
```

Comment:
- [ ] Some migrations are not working, they seem to be in the wrong order:

![image](https://user-images.githubusercontent.com/57421823/199853263-2198daad-2a73-4385-828c-1120bdd049fb.png)

Solution:

You need to recreate the migrations in the correct order.  The users table should come first, as the appointments table needs the users table to be there first.  Since the appointments table references users, this fails.

So you can do this:

```bash
rake db:migrate VERSION=0  #This will rollback all migrations
 ```

- Then, save your migration's code, so you don't have to write it again.
- Delete your 3 migrations file.
- Regenerate your migrations IN ORDER (user's table is first!).
- `rake db:migrate` to the database again.
- Your migrations will run well for you and anybody who clones your project.

Now that you know how, kindly fix your migrations, so people that clone your project are able to run it. 💪 

_Since I couldn't run your migrations, that means I couldn't run the project frontend or the tests.  This will be reviewed in the next review._